build.gradle

```diff
@@ -1,71 +1,73 @@
+buildscript {
+
+	repositories {
+		mavenLocal()
+		mavenCentral()
+		jcenter()
+		gradlePluginPortal()
+		maven { url "https://repo.spring.io/plugins-release" }
+		maven { url "https://plugins.gradle.org/m2/" }
+	}
+	allprojects {
+		repositories {
+			jcenter()
+			maven { url "https://jitpack.io" }
+		}
+	}
+
+	dependencies {
+		classpath "org.springframework.boot:spring-boot-gradle-plugin"
+		classpath "io.spring.gradle:propdeps-plugin:0.0.10.RELEASE"
+		classpath "gradle.plugin.com.gorylenko.gradle-git-properties:gradle-git-properties:1.5.1"
+		classpath "javax.activation:javax.activation-api:1.2.0"
+	}
+}
+
 plugins {
+	id 'java'
 	id "org.sonarqube" version "2.6.2"
 	id 'org.springframework.boot' version '2.3.2.RELEASE'
 	id 'io.spring.dependency-management' version '1.0.9.RELEASE'
-	id 'java'
-	id 'application'
-	id "jp.co.soramitsu.sora-plugin" version "0.3.0"
 }
 
-group = 'soramitsu'
-version = '0.0.1-SNAPSHOT'
-sourceCompatibility = '11'
+allprojects {
+	group = 'soramitsu.iroha-utils'
```

Comment:
does it conform to our naming policy?

Reply:
this is renamed by DevOPs to 'soramitsu' in master